app/src/main/java/de/westnordost/streetcomplete/quests/max_height/AddMaxHeightForm.kt

```diff
@@ -97,30 +56,12 @@ class AddMaxHeightForm : AbstractQuestFormAnswerFragment<MaxHeightAnswer>() {
     }
 
     private fun userSelectedUnrealisticHeight(): Boolean {
-        val height = getHeightFromInput() ?: return false
-        val m = height.toMeters()
-        return m > 6 || m < 1.9
+        val m = lengthInput.length?.toMeters() ?: return false
+        return m > 6 || m < 1.8
```

Comment:
Is the change from `1.9` to `1.8` deliberate?

Reply:
yes